### DISCUSSION TITLE
Wix 3.11.1 and 3.11.2 fail to install

### DISCUSSION THREAD

**Discussion by jnanneng** [wixtoolset/issues#6295] [Questions] [answered]
Please provide answers to the following questions to help us narrow down, reproduce, and fix the problem. Fill out one section and delete the others.

* Which version of WiX are you building with?
 3.11.1 and 3.11.2 fail to install.

* Which version of Visual Studio are you building with (if any)?
Microsoft Visual Studio Community 2019 Version 16.8.2

* Which version of the WiX Toolset Visual Studio Extension are you building with (if any)?
VSIX extension was downloaded from the wix toolset org website today.  v 1.0.0.4

* If the problem occurs when installing your packages built with WiX, what is the version of Windows the package is running on?
N/A, but I'm on Windows 10 pro.

* Describe the problem and the steps to reproduce it.
Ran Wix311.exe, then chose to 'install'.  That progress sprocket gets half way then indicated failed, try again?  repeats.

=> So how do I see what the failure actually is?   Log file somewhere, etc??

**Comment by rseanhall** [comment, marked as answer]
@jnanneng  The WiX Toolset does not use GitHub issues for support. I have converted this issue into a discussion item, which is an experiment that we're trying out. Alternatively, you may want to ask your question on the [wix-users mailing list](https://wixtoolset.org/documentation/mailinglist/) or [Stack Overflow](https://stackoverflow.com/questions/tagged/wix).

The log file is in %TEMP%, please post that here if you have further questions.

**Reply by jnanneng** [reply]
Interesting; the wix toolset org sent me here to log an issue.

Anyhow, what I ended up doing is noticing that the 3.11.1 exe indicated an update was available.  So I clicked that, then it installed without error.

3.11.2 now indicates that Wix is already present.

This was really odd; but Wix is now installed and running on the machine.
